docs/development/workflow/development_workflow.rst

```diff
@@ -501,6 +501,7 @@ it into Astropy:
    the maintainers know that your work is not ready for a full review nor to be
    merged yet. In addition, if your commits are not ready for CI testing, you
    should also use ``[ci skip]`` or ``[skip ci]`` directive in your commit message.
+   For usage of pre-commit hooks and directives, see :ref:`pre-commit` and :ref:`pre-commit_bot`.
```

Comment:
Ah, so it was all there – had been looking mainly under the commit message style guide in `_edit-flow`. :)

Is the draft pull request not pre-set as default now? Maybe that should be clarified as well.

Reply:
> Is the draft pull request not pre-set as default now?

I don't think so. Did they roll out a new setting and I missed it?

Reply:
I think it remembers the option you used in your last pull request via some browser cache.

Reply:
That makes sense... I'd never trust myself. ☝🏻